extensions/blocks/map/component.js

```diff
@@ -312,20 +325,22 @@ export class Map extends Component {
 			window.addEventListener( 'resize', this.debouncedSizeMap );
 		} );
 	}
-	googlePoint2Mapbox( google_point ) {
-		const mapCenter = [
-			google_point.longitude ? google_point.longitude : 0,
-			google_point.latitude ? google_point.latitude : 0,
-		];
-		return mapCenter;
-	}
+	googlePoint2Mapbox = google_point =>
+		google_point.hasOwnProperty( 'lat' ) && google_point.hasOwnProperty( 'lng' )
```

Comment:
AFAIK `hasOwnProperty` is not supported by IE11.
I guess we can't just do a truthy check (e.g. `!! google_point.lat`) as the `0` value is acceptable.
What about using lodash `has`?

Reply:
I was under the impression `hasOwnProperty` was safe everywhere? https://caniuse.com/#search=hasOwnProperty

Reply:
Shoot, you're totally right.
I went to double-check on MDN just in case, and stopped at the "Edge" column, without realizing that IE was listed separately  🤦‍♂